Dockerfile

```diff
@@ -0,0 +1,39 @@
+#TODO: How to give ability to another version and variant?
+FROM monogramm/docker-taiga-back-base:4.2-alpine
+LABEL maintainer="Monogramm maintainers <opensource at monogramm dot io>"
+
+# Taiga additional properties
+ENV TAIGA_ENABLE_LDAP=False \
+    TAIGA_LDAP_USE_TLS=True \
+    TAIGA_LDAP_SERVERS= \
+    TAIGA_LDAP_PORT=389 \
+    TAIGA_LDAP_BIND_DN= \
+    TAIGA_LDAP_BIND_PASSWORD= \
+    TAIGA_LDAP_BASE_DN= \
+    TAIGA_LDAP_USERNAME_ATTRIBUTE=uid \
+    TAIGA_LDAP_EMAIL_ATTRIBUTE=mail \
+    TAIGA_LDAP_FULL_NAME_ATTRIBUTE=cn \
+    TAIGA_LDAP_SAVE_LOGIN_PASSWORD=True \
+    TAIGA_LDAP_FALLBACK=normal
+
+# Erase original entrypoint and conf with custom one
+COPY local.py /taiga/
+COPY entrypoint.sh ./
+
+RUN set -ex ; \
+    mkdir /usr/src/taiga-contrib-ldap-auth-ext;
+
+COPY . /usr/src/taiga-contrib-ldap-auth-ext/
+
+RUN set -ex ; \
+    python /usr/src/taiga-contrib-ldap-auth-ext/setup.py bdist_wheel;
+
+COPY dist/taiga_contrib_ldap_auth_ext-0.4.4-py3-none-any.whl ./
```

Comment:
@AminovE99 Are you sure of this ?? the dist is built locally inside the docker container, not on the host. There should be nothing to copy.